x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -13,6 +13,12 @@ import { CommentRt } from './comment';
 import { CasesStatusResponseRt, CaseStatusRt } from './status';
 import { CaseConnectorRt } from '../connectors/connector';
 import { CaseAssigneesRt } from './assignee';
+import {
+  MAX_ASSIGNEES_FILTER_LENGTH,
+  MAX_REPORTERS_FILTER_LENGTH,
+  MAX_TAGS_FILTER_LENGTH,
+} from '../../constants';
+import { limitedArraySchema } from '../../schema';
```

Comment:
Not for this PR but I think we should put our `io-ts` validation functions/helpers in a different folder. Given all the conversation around schemas, types, versioned APIs, etc the name of the folder is confusing hahaha.